apps/settings/js/icc.js

```diff
@@ -61,6 +59,18 @@ require([
     var _backHandler = null;
     init();
 
+
+    function localizeElement(element, l10n) {
+      if (typeof(l10n) === 'string') {
+        element.setAttribute('data-l10n-id', l10n);
+      } else if (l10n.raw) {
+        element.removeAttribute('data-l10n-id');
+        element.textContent = l10n.raw;
+      } else {
+        navigator.mozL10n.setAttributes(element, l10n.id, l10n.attrs);
+      }
+    }
+
```

Comment:
The usage of this function and the whole l10n/string processing in this file is not intuitive to me. Could you add some comments to make it more clear? Thanks.